src/IO/lbsStatistics.py

```diff
@@ -107,6 +107,42 @@ def sampler(distribution_name, parameters, logger: Logger = None):
         logger.error(f"{distribution_name} distribution is not supported.")
         return None, None
 
```

Comment:
- 2 blank lines here
- function name `Hamming_distance` should be lowercase [PEP8 function and variable names](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
Resolve for 2 blank lines.
Please check other comment about Hamming distance.